backend/service/k8s/k8s.go

```diff
@@ -80,6 +80,9 @@ type Service interface {
 	// CronJob management functions.
 	DescribeCronJob(ctx context.Context, clientset, cluster, namespace, name string) (*k8sapiv1.CronJob, error)
 	DeleteCronJob(ctx context.Context, clientset, cluster, namespace, name string) error
+
+	// ConfigMap management functions.
+	ListConfigMaps(ctx context.Context, clientset, cluster, namespace string, listOptions *k8sapiv1.ListOptions) ([]*k8sapiv1.ConfigMap, error)
```

Comment:
To maintain our current patterns we should rename this to `DescribeConfigMap`, if you check out the other describes you will see they are all `List()` invocations. This allows us to search all namespaces, you can read more about this history here https://github.com/lyft/clutch/pull/271.